Shock/Classes/MockServer.swift

```diff
@@ -81,6 +81,9 @@ public class MockServer {
         case .collection(let routes):
             routes.forEach { self.setup(route: $0) }
             return
+        case .timeout(let method, let urlPath, let timeoutInSeconds):
+            response = responseFactory.makeResponse(urlPath: urlPath, method: method.rawValue, timeout: timeoutInSeconds)
+            break
```

Comment:
I don't think we need a break here, nor in any other case above (the `return` in `.collection` is important though).

Reply:
Yeah, copypasta...